.travis.yml

```diff
@@ -16,22 +16,22 @@ install:
   - go get github.com/mattn/goveralls
   - go get -u honnef.co/go/tools/cmd/staticcheck
 
-#  - go get -u honnef.co/go/tools/cmd/megacheck
-#  - go get -u github.com/client9/misspell/cmd/misspell
-
 before_script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then EXCLUDE_VENDOR=$(go list ./... | grep -v "/vendor/") && $(exit $(go fmt $EXCLUDE_VENDOR | wc -l)) && go vet $EXCLUDE_VENDOR; fi
   - staticcheck ./...
 
 script:
     go test -coverpkg=./... -coverprofile=./coverage.out ./...
 
-after_success:
-- git reset --hard
-
 deploy:
 - provider: script
-  skip_cleanup: true
+  script: $HOME/gopath/bin/goveralls -coverprofile=v2/coverage.out -service travis-ci
+  on:
+    tags: true
```

Comment:
So you would publish coverage only for tag pushes?

Reply:
changed to always